src/class-fields/yield-var.case

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: yield var
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  yield
```

Comment:
This example is totally fine; what those parameters control is what yield and await mean within a computed property name.

The `[?Yield, ?Await]` parameters are passed on through, where they eventually make it to an IdentifierReference. [here](https://tc39.github.io/ecma262/#prod-IdentifierReference), IdentifierReference includes `yield` and `await` only if the corresponding parameter is off.